server/views/components/search-by-crn-or-name-results/macro.njk

```diff
@@ -0,0 +1,23 @@
+{%- from "moj/components/pagination/macro.njk" import mojPagination -%}
+
+
+{% macro searchByCrnOrNameResults(params) %}
+    {% if params.errors.crn %}
+        <h2>You have not entered any search terms</h2>
+
+        <p>Enter a CRN. This can be found in nDelius.</p>
+
+    {% elif (params.resultsHtml| trim |length > 0) or (not params.crn_or_name) %}
+        
+        {{ params.resultsHtml | safe | trim | indent(8)}}
+
+    {% else %}
+        <h2>There are no results for ‘{{ params.crn_or_name }}’ in {{ params.uiStatus }} {{ params.type }}.</h2>
+
+        <p>Check the other lists.</p>
+
+        <p>If the referral is missing from every list, <a href="mailto:{{ PhaseBannerUtils.supportEmail }}">contact
+                support</a> for help.</p>
```

Comment:
This bit at least will need to refer to the correct type. Maybe a more flexible `noResultsHtml` would be ok?

Reply:
https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/1037/files#diff-07d06d7be8e214c5b7456ab398b211885a412d072012b2b92c9b3f06d30815e9R26-R35